hack/install-tls-secrets.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# Original script is here [1] we have just modify it to use ${KUBECTL} env var
+# pointing to kubevirtci kubectl binary.
+#
+# [1] https://github.com/morvencao/kube-mutating-webhook-tutorial/blob/master/deployment/webhook-create-signed-cert.sh
+
+set -e
+
+usage() {
+    cat <<EOF
+Generate certificate suitable for use with an sidecar-injector webhook service.
+
+This script uses k8s' CertificateSigningRequest API to a generate a
+certificate signed by k8s CA suitable for use with sidecar-injector webhook
+services. This requires permissions to create and approve CSR. See
+https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster for
+detailed explantion and additional instructions.
+
+The server key/cert k8s CA cert are stored in a k8s secret.
+
+usage: ${0} [OPTIONS]
+
+The following flags are required.
+
+       --service          Service name of webhook.
+       --namespace        Namespace where webhook service and secret reside.
```

Comment:
is order of parameter options relevant? use of function in cluster-sync-handler.sh seems different from this parameter order.